apps/src/templates/amazonFutureEngineerEligibility/amazonFutureEngineerAccountConfirmation.jsx

```diff
@@ -5,7 +5,7 @@ import {Button} from 'react-bootstrap'; // eslint-disable-line no-restricted-imp
 import {studio, pegasus} from '@cdo/apps/lib/util/urlHelpers';
 import color from '@cdo/apps/util/color';
 
-const RETURN_TO = `user_return_to=${pegasus('/afe/submit')}`;
+const RETURN_TO = `user_return_to=${pegasus('/afe')}`;
```

Comment:
@kelbyhawn it actually just works from changing the link here!

Reply:
Magic! ✨